smart-contracts/cookbook/smart-contracts/deploy-basic-contract-evm/deploy-basic-ethers-js.md

```diff
@@ -0,0 +1,238 @@
+---
+title: JavaScript with Ethers.js
+description: Learn how to deploy a basic smart contract to Polkadot Hub using Ethers.js, best for lightweight, programmatic deployments and application integration.
+categories: Smart Contracts
+---
+
+[Ethers.js](https://docs.ethers.org/v6/){target=\_blank} provides a lightweight approach for deploying contracts using pure JavaScript. This method is ideal for developers who want programmatic control over the deployment process or need to integrate contract deployment into existing applications.
```

Comment:
We always define an H1, even if it's the same as the title.

This is the introduction, so what is the guide about?